Update k8s app labels

matzew opened the pull request:
We use /name=eventing for all resources and use the /component to
designate different eventing components

Different resources with the same /component label contribute to
the function of that component - ie. broker-filter

Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>


(came across this via the PR https://github.com/knative/serving/pull/12247  from @dprotaso )

matzew commented:
/hold
/assign @devguyio 
/assign @lionelvillard

matzew commented:
/test pull-knative-eventing-integration-tests

matzew commented:
/unhold

devguyio commented:
sorry for getting late to this, just to be safe I'm gonna hold it to make sure this won't break anything, I vaguely remember that changing those labels was problematic before but I might be mistaken 
/hold

matzew commented:
I fine in not doing this - but I personally found it odd, that serving is (again) different here.
I noticed by accident, and therefore performed the same changes, the description seems reasonable to me.

Serving:
- introduced on 0.25
- changed for the 1.1 _upcoming_ release (see linked PR)

Eventing:
- introduced on 0,26 (not sure why not aligned with serving for doing it for 0.25 too)
- changed for the 1.1 _upcoming_ release (this PR)

devguyio commented:
/lgtm
leaving it to @lionelvillard to unhold

lionelvillard commented:
/unhold
